documentation/reference/tensor.html

```diff
@@ -294,8 +360,12 @@ <h4 id="tensor">- tensor</h4>
 Examples:
 {% raw %}
 <pre>
-tensor&lt;float&gt;(x[3])(x) = {{x:0}: 0.0, {x:1}: 1.0, {x:2}: 2.0}
+t1 = tensor(x[4]):[1.0, 2.0, 3.0, 4.0]
+
+tensor&lt;float&gt;(x[3])(x) = [0.0, 1.0, 2.0]
 tensor&lt;float&gt;(x[2],y[2])(x == y) = {{x:0,y:0}: 1.0, {x:0,y:1}: 0.0, {x:1,y:0}: 0.0, {x:1,y:1}: 1.0}
+tensor(f[2])(t1{x: f + f()})) - Create a size 2 1d tensor by accessing t1 at two indexes given by adding
+                                the index of it (f - which is 0 and 1) by a function also named f.
```

Comment:
This example is perhaps a bit dense. I think the concept of generating a tensor based on another is sufficiently complex that it warrants an expanded description, with showing a few different examples with varying complexity.

Reply:
Agreed, just wanted to capture the f and f() thing *somewhere* for now.